Production_scripts/Process_extracts/extract_alarm_telecare.R

```diff
@@ -0,0 +1,228 @@
+  tidylog::mutate(
+    start_date_missing = is.na(service_start_date),
+    service_start_date = if_else(
+      start_date_missing,
+      start_fy(year = substr(period, 1, 4), format = "alternate"),
+      service_start_date
+    )
+  )
+
+at_full_clean <- replaced_start_dates %>%
+  # Match on demographics data (chi, gender, dob and postcode)
+  left_join(sc_demographics, by = c("sending_location", "social_care_id")) %>%
+  # rename for matching source variables
+  rename(
+    record_keydate1 = service_start_date,
+    record_keydate2 = service_end_date
+  ) %>%
+  # Include source variables
+  mutate(
+    recid = "AT",
+    smrtype = case_when(
+      service_type == 1 ~ "AT-Alarm",
+      service_type == 2 ~ "AT-Tele"
+    ),
+    # Create person id variable
+    person_id = glue::glue("{sending_location}-{social_care_id}"),
+    # Use function for creating sc send lca variables
+    sc_send_lca = convert_sending_location_to_lca(sending_location)
+    # Changes here compared with 03-Alarms_Telecare_data.R
+    # convert_sending_location_to_lca against convert_sc_to_lca
+  ) %>%
+  # when multiple social_care_id from sending_location for single CHI
+  # replace social_care_id with latest
+  group_by(sending_location, chi) %>%
+  mutate(latest_sc_id = last(social_care_id)) %>%
+  # count changed social_care_id
+  mutate(
+    changed_sc_id = !is.na(chi) & social_care_id != latest_sc_id,
+    social_care_id = if_else(changed_sc_id, latest_sc_id, social_care_id)
+  ) %>%
+  ungroup()
+
+# Deal with episodes which have a package across quarters.
+qtr_merge <- at_full_clean %>%
+  # Use lazy_dt() for faster running of code
+  dtplyr::lazy_dt() %>%
+  group_by(
+    sending_location,
+    social_care_id,
+    record_keydate1,
+    smrtype,
+    period
+  ) %>%
+  # Create a count for the package number across episodes
+  mutate(pkg_count = row_number()) %>%
+  # Sort prior to merging
+  arrange(.by_group = TRUE) %>%
+  # group for merging episodes
+  group_by(
+    sending_location,
+    social_care_id,
+    record_keydate1,
+    smrtype,
+    pkg_count
+  ) %>%
+  # merge episodes with packages across quarters
+  # drop variables not needed
+  summarise(
+    sending_location = last(sending_location),
+    social_care_id = last(social_care_id),
+    sc_latest_submission = last(period),
+    record_keydate1 = last(record_keydate1),
+    record_keydate2 = last(record_keydate2),
+    smrtype = last(smrtype),
+    pkg_count = last(pkg_count),
+    chi = last(chi),
+    gender = last(gender),
+    dob = last(dob),
+    postcode = last(postcode),
+    recid = last(recid),
+    person_id = last(person_id),
+    sc_send_lca = last(sc_send_lca)
+  ) %>%
+  # sort after merging
+  arrange(
+    sending_location,
+    social_care_id,
+    record_keydate1,
+    smrtype,
+    sc_latest_submission
+  ) %>%
+  # end of lazy_dt()
+  as_tibble() %>%
+  # Sort for running SPSS
+  arrange(
+    sending_location,
+    social_care_id
+  )
```

Comment:
This should be in a separate script - only the below is dependant on year

We will have 2 scripts; This code should be in `"Production_scripts/Social_Care/alarm_telecare.R"` and the year specific code will be in `"Production_scripts/Process_extracts/extract_alarm_telecare.R"`